tests/mock_smtp.py

```diff
@@ -70,7 +70,7 @@ def loop(self):
         exit.
         """
         while len(asyncore.socket_map):
-            asyncore.loop(timeout=1, use_poll=True)
+            asyncore.loop(timeout=0, use_poll=True)
```

Comment:
There was another way to squelch those errors (since they are just about having threads stopped in a sub-optimal order).  I'll have to check what I was doing several months ago...